.circleci/config.yml

```diff
@@ -0,0 +1,37 @@
+# Javascript Node CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-javascript/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      # specify the version you desire here
+      - image: circleci/node:7.10
```

Comment:
Let's do node 8 here

Reply:
So just `circleci/node:8`?

Reply:
Yes, that should work. 

Jest uses `image: circleci/node:10` so just replacing it with 8 should work